.gitignore

```diff
@@ -7,3 +7,4 @@ go.work.sum
 .github/artifacts/data.json
 cover.out
 coverage.txt
+test-results/
```

Comment:
I saw that one junit file got committed below, so might need a bit of an update here.

Reply:
Okay I will make the changes, good catch.

Reply:
I had changed the path from `test-results` to `test-output` so thats why that file got comitted. It was an old one, deleted in [this commit](https://github.com/scality/mountpoint-s3-csi-driver/pull/13/commits/957291ac53c24fe399038707a0bba15b2f97c3e8)

Tests use `test-results/` so no update needed in `.gitignore`. :)